### ISSUE TITLE
[QUESTION] Do you know if the plugin is working with Ckan 2.11 ?

### ISSUE THREAD

**Issue by BENETNATH** [keitaroinc/ckanext-password-policy#12]
Title should be self explanatory
I'm not the admin of the instance, so I'd like to know if we could implement a password policy, to raise the security level.
But before asking, I'd love to know if that could work,
thanks

**Comment by blagojabozinovski** [comment]
@BENETNATH Yes, the extension has been updated to be compatible with CKAN 2.11. You’re welcome to implement it, report any issues you encounter, or contribute improvements to the project.
